global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "projects": [ "src", "." ],

```

Comment:
Does the `.` folder need to be listed? All the source projects are in the `src` folder as far as I can tell.